src/python/grpcio_tests/tests_aio/unit/channel_test.py

```diff
@@ -39,16 +42,34 @@ def test_async_context(self):
     def test_unary_unary(self):
 
         async def coro():
-            channel = aio.insecure_channel(self.server_target)
-            hi = channel.unary_unary(
-                '/grpc.testing.TestService/UnaryCall',
-                request_serializer=messages_pb2.SimpleRequest.SerializeToString,
-                response_deserializer=messages_pb2.SimpleResponse.FromString)
-            response = await hi(messages_pb2.SimpleRequest())
+            async with aio.insecure_channel(self.server_target) as channel:
```

Comment:
by doing this you remove the test that checks the explicit call of `aio.insecure_channel` without a context manager. Note that above there is a test for context manager.

I would at least leave one test that checks for the explicit call

Reply:
Reverted 👍